src/main/resources/application.conf

```diff
@@ -44,7 +44,13 @@ nav {
             alias = "login-service-v1"
             discovery_endpoint = ""
             discovery_endpoint = ${?LOGIN_SERVICE_V1_DISCOVERY_ENDPOINT}
-        }, {
+        },
+        {
+            alias = "login-service-v2"
```

Comment:
Bør legge til `audience` også. Ser vi ikke har hatt den før på v1. Men bør finne hva audiencen er for v1 og confe den, mens for v2 var vel det en annen env-variable de tilgjengeligjorde.

Reply:
Den er grei 👍